CHANGES/8267.feature

```diff
@@ -0,0 +1 @@
+The `pulp_content_workers` option can be used to adjust the number of Gunicorn worker processes handling content app requests.
```

Comment:
Update this for pulp_api_workers as well.

Reply:
That was already present, it's not a new feature.  I only touched the documentation on that one.